app/views/users/profile.html.erb

```diff
@@ -129,38 +131,45 @@
 
     <% if @profile_user && logged_in_as(['admin']) && @profile_user.role != "admin" %>
       <a href="/admin/promote/admin/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-certificate"></i> <%= t('users.profile.make_admin') %></a>
+      <hr />
     <% end %>
 
     <% if @profile_user && logged_in_as(['admin']) && @profile_user.role == "moderator" %>
       <a href="/admin/demote/basic/<%= @profile_user.id %>" class="btn btn-default btn-block"><i class="fa fa-ban"></i> <%= t('users.profile.revert_to_basic_user') %></a>
+      <hr />
     <% end %>
 
     <% if @profile_user && logged_in_as(['admin', 'moderator']) && @profile_user.role == "basic" %>
       <a href="/admin/promote/moderator/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-certificate"></i> <%= t('users.profile.make_moderator') %></a>
+      <hr />
     <% end %>
 
     <% if logged_in_as(['admin', 'moderator']) && @profile_user.status == 0 %>
       <a href="/unban/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-thumbs-up"></i> <%= t('users.profile.unban_user') %></a>
+      <hr />    
     <% elsif logged_in_as(['admin', 'moderator']) && @profile_user.status == 1 && @profile_user != current_user %>
       <a data-confirm="<%= t('users.profile.confirm_ban_user') %>" href="/ban/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-ban"></i> <%= t('users.profile.ban_user') %></a>
+      <hr />
     <% end %>
 
     <% if logged_in_as(['admin', 'moderator']) && @profile_user.status == 5 %>
       <a href="/admin/unmoderate/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-thumbs-up"></i> <%= t('users.profile.unmoderate_user') %></a>
+      <hr />
     <% elsif logged_in_as(['admin', 'moderator']) && @profile_user.status == 1 %>
       <a rel="tooltip" title="Prevent author from logging in, without hiding their work or profile." data-confirm="<%= t('users.profile.confirm_moderate_user') %>" href="/admin/moderate/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-ban"></i> <%= t('users.profile.moderate_user') %></a>
+      <hr />
     <% end %>
 
     <% if @profile_user && !@profile_user.reset_key.nil? && logged_in_as(['admin']) %>
       <a id="user-reset-key" href="/reset/key/<%= @profile_user.reset_key %>" class="btn btn-outline-secondary btn-block"><i class="fa fa-info-sign"></i> <%= t('users.profile.reset_user') %>: <%= @profile_user.reset_key %></a>
+      <hr />
     <% end %>
 
     <% if @profile_user && @profile_user.reset_key.nil? && logged_in_as(['admin', 'moderator']) && (@profile_user.role == "basic" || @profile_user.role == "moderator") %>
       <a href="/admin/force/reset/<%= @profile_user.id %>" class="btn btn-outline-secondary btn-block">Reset User Password</a>
+      <hr />
```

Comment:
So the way this is set up now is that there will be a HR shown for each of those items, an admin would see 7 of them. We only one one HR at the bottom, but we only want it to show if something else is showing. Maybe we could set a ruby variable up at line 90, something like logged_content = false. Then in every one of those blocks instead of showing `<hr />' you would set the variable to true. Then at the bottom only display the hr `if logged_content == true`

Reply:
@nstjean 
while using the logged_content at the bottom it encounters an error showing "Undefined local variable logged_content", even after declaring it. I am stuck here do you have any solution.

Reply:
@NitinBhasneria Can you share that chunk of code?